rpc/src/module/chain.rs

```diff
@@ -1191,6 +1191,50 @@ pub trait ChainRpc {
     /// ```
     #[rpc(name = "get_block_median_time")]
     fn get_block_median_time(&self, block_hash: H256) -> Result<Option<Timestamp>>;
+
+    /// Returns the median fee rate in recent number of blocks specified by user.
+    ///
+    /// ## Params
+    ///
+    /// * `max_blocks_scan` - the latest x (default: 42) blocks scanned to calculate median fee rate.
+    /// * `min_transaction` - minimal number transactions(default: 42) included in the block.
+    ///
+    /// ## Returns
+    /// * `Result<Capacity>` - the median fee rate(in Shannons).
+    ///
+    /// When the `blocks_to_scan` is greater than `max_blocks_to_scan(default: 42)`, this RPC returns error;
+    /// otherwise returns the median fee rate (in Shannons).
+    ///
+    /// ## Examples
+    ///
+    /// Request
+    ///
+    /// ```json
+    /// {
+    ///   "id": 42,
+    ///   "jsonrpc": "2.0",
+    ///   "method": "get_median_fee_rate",
+    ///   "params": [
+    ///     "0x10"
```

Comment:
Tweak the params. There's no enough transactions in the RPC test chain. Try 1 transaction in the last 11 blocks and paste the corresponding result below.